flambe/field/text.py

```diff
@@ -85,6 +87,10 @@ def __init__(self,  # nosec
             Whether to drop tokens that don't have embeddings
             associated. Defaults to True.
             Important: this flag will only work when using embeddings.
```

Comment:
(Out of scope but lets try to add it)

Guys wee are missing the `truncate_end` doc. @jeremyasapp can you provide a quick reference on what it does so that we can include?

Reply:
I didn't add that, not sure what it does.